editor/src/clj/editor/asset_browser.clj

```diff
@@ -814,12 +814,26 @@
         dropped-handler (ui/event-handler e (error-reporting/catch-all! (drag-dropped e)))
         detected-handler (ui/event-handler e (drag-detected e (handler/selection selection-provider)))
         entered-handler (ui/event-handler e (drag-entered e))
-        exited-handler (ui/event-handler e (drag-exited e))]
+        exited-handler (ui/event-handler e (drag-exited e))
+        original-dispatcher (.getEventDispatcher tree-view)]
     (doto tree-view
       (ui/customize-tree-view! {:double-click-expand? true})
-      (ui/bind-double-click! :open)
-      (ui/bind-key-commands! {"Enter" :open
-                              "F2" :rename})
```

Comment:
We used to force `F2` here to work around the TreeView F2 handling that we now workaround differently below. Both F2 and other shortcuts for renaming work now!